File: foundation 2 month/lesson6/homework/ex_5.c
#include <stdio.h>
#include <stdlib.h>

void check( char ms[],char ms1[])
{
	int i,j;
	int cnt=0;
	for(i=0; ms[i] !='\0';i++)
	{
		for(j=0; ms1[j] !='\0';j++)
		{
		if(ms[i]==ms[j])
		{
		cnt++;
		}
		}
	}
	if(i==cnt)
	printf("Bil xil");
	else
	puts("bir xil emas");
	puts("");

}

int main()
{
	char  str[100];
	char str2[100];
	printf("1 chi so'zni kiriting: ");
	scanf("%[^\n]%*c",str);
        printf("2 chi so'zni kiriting: ");
        scanf("%[^\n]%*c",str2);
	check(str,str2);
}

File: foundation 3 month/lesson1/homework/ex_5.py
def check(s):
	m=s.split()
	cnt=0
	ms=list()
	for x in m:
		for y in x:
			if y in '0123456789':
				y='#'
				cnt+=1
				ms.append(y)
			else:
				ms.append(y)
		ms.append(' ')
	if cnt==0:
		print(s)
	else:
		for x in ms:
			print(str(x),end='')

k=input("satrni kiriting: ")
check(k)
